setup.py

```diff
@@ -1,4 +1,6 @@
-import os, sys
+import os
+import sys
```

Comment:
I don't really care, but what's the rationale between having an explicit single line import on these two lines and having multiple imports on one line for the other places? 

Again, it's not a big deal, I'm just curious about reasons for the choice to change it.